lib/suppressed-errors.js

```diff
@@ -52,7 +52,9 @@ async function write(options, suppressionFiles) {
 
       const previousContents = await FS.readFile(filePath).catch(() => '');
       if (previousContents !== newContents) {
-        return FS.writeFile(filePath, newContents).catch((error) => {
+        try {
+          return await FS.writeFile(filePath, newContents);
```

Comment:
What is the value of adding `await` after a `return`? :thinking:

Reply:
Okay, the rule description gives some arguments in favor: https://typescript-eslint.io/rules/return-await/
Alright, let's try it for the sake of better stack traces :+1:

Reply:
It's faster (you'll see people saying it's slower, but it's now better optimized), it has better stack traces, and it makes try-catch work.